PyPowerFlex/objects/sdc.py

```diff
@@ -14,8 +14,9 @@
 # under the License.
 
 import logging
-
+import requests
```

Comment:
the request library is not used here. can we remove this?

Reply:
We can fix across modules in the upcoming release.